modules/mcagar/src/main/ts/ephox/mcagar/api/Settings.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
This file shouldn't be here, as these aren't part of the McAgar API. They should probably just remain as is.

Reply:
Alright.